utils/useragent/useragent.go

```diff
@@ -0,0 +1,30 @@
+package useragent
```

Comment:
Can we add tests for this wrapper using samples of the user agent for Brave on each platform?

Reply:
fixed in: 49fdac0